packages/cloud-functions/.gitignore

```diff
@@ -0,0 +1 @@
+*/yarn.lock
```

Comment:
This seems like you only needed to ignore the `yarn.lock` locally, probably shouldn't go in this PR

Reply:
main reason I had this is that I dont want the yarn lock files to be included within any of the sub packages of the `cloud-functions`. The lock file will be produced when building locally but I dont want this to be included in the repo for these scripts seeing they tend to have only one or two dependencies.